portfolio/src/main/webapp/script.js

```diff
@@ -28,3 +28,9 @@ function addRandomQuote() {
   const quoteContainer = document.getElementById('quote-container');
   quoteContainer.innerText = quote;
 }
+
+function getMyNameUsingArrowFunctions() {
```

Comment:
for naming functions, it's usually clearer to just use something that describes the function's purpose like getMyName() instead of something that describes both the purpose and how it's done like getMyNameUsingArrowFunctions(). the one exception that i personally use to this is if you have two functions that do the same thing but in different ways, then i would use a naming scheme like this to make the difference clear.